woocommerce/woocommerce-android
# [Deposit Summary] Deposit Summary View to the payments hub

Closes: #10068


### Description

Adds deposit summary view to the Payments Hub screen. The view itself is not finalized yet and hidden behind feature flag.

### Testing instructions

* Make sure that the view is not shown when the FF is off (in release)

Don't pay much attention to the UI itself as it's in progress and will be addressed in the following PRs

### Images/gif

<img width="368" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/918b64d4-1aa7-4e39-9100-4ad3accaa02a">
